src/build/arg/arg_value.rs

```diff
@@ -0,0 +1,71 @@
+use std::fmt::Display;
+
+/// TODO
+#[derive(Debug, Default, Clone)]
+pub struct ArgValue<'help> {
+    pub(crate) value: &'help str,
+    pub(crate) about: Option<&'help str>,
+    pub(crate) hidden: bool,
+}
+
+impl Display for ArgValue<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.value.contains(char::is_whitespace) {
+            write!(f, "{:?}", self.value)
+        } else {
+            write!(f, "{}", self.value)
+        }
+    }
+}
+
+impl<'help> From<&'help str> for ArgValue<'help> {
+    fn from(s: &'help str) -> Self {
+        Self::new(s)
+    }
+}
+
+/// Getters
+impl<'help> ArgValue<'help> {
+    /// TODO
+    #[inline]
+    pub fn get_value(&self) -> &str {
+        self.value
+    }
+
+    /// TODO
+    #[inline]
+    pub fn get_about(&self) -> Option<&str> {
+        self.about
+    }
+
+    /// TODO
+    #[inline]
+    pub fn get_hidden(&self) -> bool {
+        self.hidden
+    }
+}
+
+impl<'help> ArgValue<'help> {
+    /// TODO
+    pub fn new<S: Into<&'help str>>(v: S) -> Self {
+        let value = v.into();
+        ArgValue {
+            value,
+            ..Default::default()
+        }
+    }
+
+    /// TODO
+    #[inline]
+    pub fn about<S: Into<&'help str>>(mut self, a: S) -> Self {
+        self.about = Some(a.into());
+        self
+    }
+
+    /// TODO
+    #[inline]
+    pub fn hidden(mut self, h: bool) -> Self {
+        self.hidden = h;
```

Comment:
Nit: This is what clap generally does but I wonder if we should follow what other crates do and name this `yes`

Reply:
you mean the parameter name?

Reply:
Yes, sorry for being unclear